scripts/encrypt/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "encrypt",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.ts",
+  "scripts": {
+    "encrypt": "ts-node index.ts"
+  },
+  "keywords": [],
+  "dependencies": {
+    "@opensystemslab/planx-core": "git+https://github.com/theopensystemslab/planx-core#ce539b0",
```

Comment:
I'd vote that we don't worry about keeping this up to date regularly. It's only using one exported function and we can update this more ad-hoc than the other projects without consequence.